storage/innobase/xtrabackup/test/inc/common.sh

```diff
@@ -445,6 +486,92 @@ function shutdown_server()
     shutdown_server_with_id 1
 }
 
+########################################################################
+# Start GR cluster
+########################################################################
+function start_group_replication_cluster()
+{
+  local number_of_nodes=$1
+  shift
+  if [[ -z  $number_of_nodes ]];
+  then
+    number_of_nodes=2
+  fi
+  bulk_init_gr_variables ${number_of_nodes}
+  for ((i=1; i<=number_of_nodes; i++))
+  do
+    switch_server ${i}
+    GR_CLUSTER_ADDRESS=""
+    for SRV in "${SRV_MYSQLD_GR_PORT[@]}"
+    do
+      GR_CLUSTER_ADDRESS="${GR_CLUSTER_ADDRESS}127.0.0.1:${SRV},"
+    done
+    MYSQLD_EXTRA_MY_CNF_OPTS="
+gtid_mode=ON
+enforce_gtid_consistency=ON
+plugin_load_add='group_replication.so'
+loose-group_replication_group_name=\"aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa\"
+loose-group_replication_start_on_boot=off
+loose-group_replication_bootstrap_group= off
+loose-group_replication_recovery_use_ssl=ON
+loose-group_replication_recovery_get_public_key=ON
+loose-group_replication_local_address=\"127.0.0.1:${SRV_MYSQLD_GR_PORT[$i]}\"
+loose-group_replication_group_seeds= \"${GR_CLUSTER_ADDRESS: : -1}\""
```

Comment:
could you use loose-group_replication_group_seeds= \"${GR_CLUSTER_ADDRESS%?}\""
:: is not working on Darwin.